src/utils/api.js

```diff
@@ -0,0 +1,53 @@
+import fetch from "node-fetch";
```

Comment:
node recently shipped built-in fetch support. Not sure if netlify supports the latest or not, just calling it out in case we can cut this dep

Reply:
Node LTS is 16.18 which doesn't have native fetch (it's in 18.0 and above), so I'm going to keep this as a dependency.